server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/DefaultMirroringServicePlugin.java

```diff
@@ -50,6 +50,7 @@ public synchronized CompletionStage<Void> start(PluginContext context) {
             mirroringService = new DefaultMirroringService(new File(cfg.dataDir(), "_mirrors"),
                                                            context.projectManager(),
                                                            context.meterRegistry(),
+                                                           context.internalProjectInitializer(),
```

Comment:
Can't we just wait here instead of passing the initializer?
The thread that invokes `internalProjectInitializer.whenInitialized().get();` in `migrate()` isn't changed.

Reply:
I wanted to perform the logic in the `MirroringMigrationService` since `MirroringMigrationService` will be removed in the future.

Anyway, either is fine.

Reply:
Or, how about calling `internalProjectInitializer.whenInitialized().get()` in  `PluginGroupStartStop.doStart()` to prevent the situation when another plugin is added?

Reply:
+1 Good idea.